docs/book/rest-api.md

```diff
@@ -1367,6 +1367,7 @@ seccomp_unconfined {
 - **pretty** - If parameter is `true`, response will formatted for humans.
 - **explain** - Return query explanation in addition to result. Values: **full**, **truth**.
 - **metrics** - Return query performance metrics in addition to result. See [Performance Metrics](#performance-metrics) for more detail.
+- **watch** - Set a watch on the data reference if the parameter is present. All other parameters are ignored if `watch` is present. See [Watches](#watches) for more detail.
```

Comment:
nope it was just simple. I'll expand watches to support metrics and tracing, and add the flags to the server's handling of them